### DISCUSSION TITLE
Dashboard/ Kalender settings

### DISCUSSION THREAD

**Discussion by schreckag** [kimai/kimai#2267] [Questions & Answers]
Hey
I am trying to customise the dashboard and calendar view.

1. where the kimai.widgets are located?
2. is it possible to show scheduled tasks in the calendar?

Thanks

**Comment by kevinpapst** [comment]
Please explain 1 - I don't really understand that question. Check these links:
- https://github.com/kevinpapst/kimai2/blob/master/config/packages/kimai.yaml#L184
- https://www.kimai.org/documentation/dashboard.html

2. If you are talking about the Task plugin, no this is not possible yet. But sounds like a nice idea. You can add a feature request to the repository of the plugin.

**Reply by schreckag** [reply]
1.
https://www.kimai.org/documentation/dashboard.html

-> wigdets

I assumed that kimai.widgets was a file. My question was where to find it. Link 1 helps me for now

2. I will gladly do that

**Reply by kevinpapst** [reply]
Being able to configure the dashboard per user in Kimai is on the roadmap
I already played around with some possible solutions, but that is a very complex topic. 
So for now, even though it is possible, its hard to change the dashboard by yourself.

**Reply by schreckag** [reply]
thx for that info. I'll test the options to do this for now.
